modules/node/src/admin/admin.provider.ts

```diff
@@ -38,10 +38,18 @@ class AdminMessaging extends AbstractMessagingProvider {
     return await this.adminService.getChannelStates(userPublicIdentifier);
```

Comment:
should probably rename this to `getChannelStateByUserId` or something more descriptive